### PR TITLE
🐛 FIX: various horizontal scrolling issues on mobile

### PR DESCRIPTION
**Pull request by violoncelloCH**

### CODE REVIEW

**Comment by violoncelloCH** [comment]
> Run `npm run lint:autofix` :)

done :)

File: src/components/FolderContent.vue [hunk 1]
```diff
@@ -13,7 +13,7 @@
 				/>
 				<NewMessageDetail v-if="newMessage" />
 				<Message v-else-if="showMessage" />
-				<NoMessageSelected v-else-if="hasMessages" />
+				<NoMessageSelected v-else-if="hasMessages && !this.isMobile" />
```
Comment (ChristophWurst) [inline src/components/FolderContent.vue]:
```suggestion
				<NoMessageSelected v-else-if="hasMessages && !isMobile" />
```

No need for `this` in a few template. It's added automagically ;)

**Comment by ChristophWurst** [comment]
How do I test this?

File: src/components/FolderContent.vue (hunk 1 above)
Comment (violoncelloCH) [reply src/components/FolderContent.vue]: whoops, of course; you're right...

**Comment by violoncelloCH** [comment]
hmm, best would be to test this together with nextcloud/server#17843 and nextcloud/nextcloud-vue#711 because those two solve related problems... or what do you mean by "how"?

**Comment by ChristophWurst** [comment]
More like what is the behavior before and after this fix :)

If we need https://github.com/nextcloud/nextcloud-vue/pull/711 then we should wait for the next release of that lib

**Comment by violoncelloCH** [comment]
test it on narrow mobile layouts of 340px width to compare before and after... 
it makes the composer fit into those 340px and and also the message list is optimized for mobile..
yeah maybe it makes sense to wait for that release; on the other hand it's more an additional improvement than a requirement for this PR

**Comment by violoncelloCH** [comment]
the only "functional" change is that it makes sure to not display the "no message selected" section on mobile (together with nextcloud/nextcloud-vue#711 this uses the correct breakpoint)...

**Pull request merged**